manifests/unit_file.pp

```diff
@@ -80,7 +83,8 @@
   Optional[String]                         $restart   = undef,
   Boolean                                  $selinux_ignore_defaults = false,
   Hash[String[1], Any]                     $service_parameters = {},
-  Boolean                                  $daemon_reload = true
+  Boolean                                  $daemon_reload = true,
+  Boolean                                  $service_restart = true
```

Comment:
Let's add the trailing comma so that we don't keep having unnecessary diffs.

```suggestion
  Boolean                                  $service_restart = true,
```

Reply:
Ah right, should've done that in the first place as it's already this way in manage_unit.pp